### PR TITLE
Upgrade PowerShell worker to 2.0.448

### LINKED ISSUE #6471: Upgrade PowerShell worker to v2.0.448 (Functions v2 only)
https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v2.0.448

### PR DESCRIPTION
**Pull request by AnatoliB**
### Issue describing the changes in this PR

resolves #6471

### Pull request checklist

* [X] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
    * [X] Otherwise: I've added my notes to `release_notes.md`
* [X] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [X] I have added all required tests (Unit tests, E2E tests)

### Additional information

PowerShell worker release notes: https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v2.0.448

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### Release notes
+<!-- Please add your release notes in the following format:
+- My change description (#PR)
+ -->
+
+- Upgrade PowerShell Workers to 2.0.448 ([Release Notes](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.448))
```
Comment (Francisco-Gamino) [inline release_notes.md]: Should the link to the release notes be `https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v2.0.448`?

**Review by Francisco-Gamino — DISMISSED** [review]
Please update the link to the release notes. Otherwise, LGTM.

File: release_notes.md (hunk 1 above)
Comment (AnatoliB) [reply release_notes.md]: Good catch, thank you.

**Review by Francisco-Gamino — APPROVED** [review]
LGTM

**Pull request merged**